frontend/src/metabase/xray/components/InsightCard.jsx

```diff
@@ -0,0 +1,368 @@
+    static icon = "warning"
+
+    render() {
+        return (
+            <InsightText>
+                Your data looks to be <TermWithDefinition definition={multimodalDefinition} link={multimodalLink}>
+                    multimodal
+                </TermWithDefinition>. This is often the case when different segments of data are mixed together.
+            </InsightText>
+        )
+    }
+}
+
+export class OutliersInsight extends Component {
+    static insightType = "outliers"
+    static title = "Outliers"
+    static icon = "warning"
+
+    render() {
+        const { filter, features: { table } } = this.props
+
+        const viewAllRowsUrl = table && Question.create()
+              .query()
+              // imitate the required hydrated metadata format
+              .setTable({ ...table, database: { id: table.db_id }})
+              .addFilter(filter)
+              .question()
+              .getUrl()
+
+        // construct the question with filter
+	return (
+		<InsightText>
+                You have some outliers.
+                <span> </span>
+                { table && <span><Link to={viewAllRowsUrl}>View all rows</Link> with outliers.</span> }
+	    </InsightText>
+        )
+    }
+}
+
+export class StructuralBreaksInsight extends Component {
+    static insightType = "structural-breaks"
+    static title = "Structural breaks"
+    static icon = "insight"
+
+    render() {
+	const { breaks, features: { resolution } } = this.props
+
+	const breakPoints = breaks.map( (point, idx) => {
+	    point = formatTimeWithUnit(point, resolution);
+	    if (idx == breaks.length - 1 && breaks.length > 1) {
+		return (
+			<span>, and { point }</span>
+		)
+	    } else {
+		return (
+			<span>{ idx > 0 && <span>, </span>}{ point }</span>
+		)
+	    }
+	})
+
+        return (
+            <InsightText>
+		It looks like your data has
+	    { breaks.length > 1 && <span> structural breaks </span>}
+	    { breaks.length == 1 && <span> a structural break </span>}
+	    at { breakPoints }.
+            </InsightText>
+        )
+    }
+}
+
+const stationaryDefinition = "Mean does not change with time."
+const stationaryLink = "https://en.wikipedia.org/wiki/Stationary_process"
+
+export class StationaryInsight extends Component {
+    static insightType = "stationary"
+    static title = "Stationary data"
+    static icon = "insight"
+
+    render() {
+
+	return (
+		<InsightText>
+                Your data looks to be <TermWithDefinition definition={stationaryDefinition} link={stationaryLink}>
+                    stationary
+                </TermWithDefinition>.
+            </InsightText>
+        )
+    }
+}
+
+export class TrendInsight extends Component {
+    static insightType = "trend"
+    static title = "Trend"
+    static icon = "insight"
+
+    render() {
+	const { mode, shape } = this.props
+
+	return(
+		<InsightText>
+		Your data seems to be { mode } { shape }.
+		</InsightText>
+	)
+    }
+}
+
+/*
```

Comment:
I'd like to remove this commented out insight if we're not shipping it as part of this batch.